pkg/api/v1/types.go

```diff
@@ -1328,6 +1337,8 @@ type NodeSystemInfo struct {
 	ContainerRuntimeVersion string `json:"containerRuntimeVersion" description:"Container runtime version reported by the node through runtime remote API (e.g. docker://1.5.0)"`
 	// Kubelet version reported by the node
 	KubeletVersion string `json:"kubeletVersion" description:"Kubelet version reported by the node"`
+	// Open ports on Kubelet
+	KubeletPorts KubeletPorts `json:"kubeletPorts" description:"Information about ports open by Kubelet"`
```

Comment:
Doesn't belong in NodeSystemInfo. Please put this just under the Addresses field in NodeStatus.